sources/packages/backend/apps/db-migrations/src/sql/SFASRestrictionMaps/Create-sfas-restriction-maps.sql

```diff
@@ -0,0 +1,31 @@
+CREATE TABLE sims.sfas_restriction_maps (
+  id SERIAL PRIMARY KEY,
+  legacy_code VARCHAR(4) NOT NULL,
+  code VARCHAR(10) NOT NULL,
+  is_legacy_only BOOLEAN NOT NULL,
+  -- Audit columns
+  created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
```

Comment:
Minor. So we see any value in creating `created_at`. For `sims.announcements` we do not have this column. so asking this question.

Reply:
It can help to audit when the restriction was added (for troubleshooting). It can also be tracked in some extension by the date that the restriction was inserted in the `sims.restrcitions`, but troubleshooting can be helpful.